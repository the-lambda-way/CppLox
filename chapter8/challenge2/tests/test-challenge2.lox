// No initializers.
var a;
var b;

a = "assigned";
print a; // OK, was assigned first.

print b; // Error!