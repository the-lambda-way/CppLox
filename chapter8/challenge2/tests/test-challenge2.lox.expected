assigned
Variable not initialized.
[line 8]
